lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -25,6 +25,8 @@ $ErrorActionPreference = "Stop"
 $params = Parse-Args $args;
```

Comment:
The trailing semi-colon is not needed, but here it would be nice if we could add check-mode support.
See the win_shortcut plugin for the exact syntax to add this to your module.

In check-mode we would like the plugin to follow as much code-paths or failure mode as possible without making an effective change. So some modules include the connection/disconnections as part of the check-mode logic, or all possible pre-flight checks a module has.

We want the module to fail exactly the same way it would fail in real run-mode. Except without making the change. In some cases we simply add the -WhatIf option to the command to let Powershell do as much as possible without the actual change.